crates/core/src/db/commit_log.rs

```diff
@@ -248,7 +245,7 @@ impl CommitLogView {
     /// the transactions in a [`Commit`].
     ///
     /// The iterator attempts to read each large object in turn, yielding an
-    /// [`io::Error`] with kind [`io::ErrorKind::NotFound`] if the object was
+    /// [`io::Error`] with constraints [`io::ErrorKind::NotFound`] if the object was
```

Comment:
Was this a stray find-and-replace change? It reads oddly now.

Reply:
You are right. I make a wrong refactor.